tests/test_transformations.py

```diff
@@ -127,6 +160,218 @@ class AllStrict(Mapping):
             expected = {"obj": {"target": 1}}
             self.assertEqual(expected, result)
 
+    def test_all_transformation_strict(self):
+        class User(Schema):
+            name = UnicodeField()
+
+        class Message(Schema):
+            id = IntegerField()
+            user = Subschema(User)
+
+        class Transformed(Schema):
+            user_name = UnicodeField()
+            object = ObjectField()
+
+        class SomeNonStrictMapping(Mapping):
+            source_schema = Message
+            target_schema = Transformed
+
+            user_name = Get('user', 'name')
+            object = All(strict=False)
+
+        class SomeStrictMapping(Mapping):
+            source_schema = Message
+            target_schema = Transformed
+
+            user_name = Get('user', 'name')
+            object = All(strict=True)
+
+        non_strict = SomeNonStrictMapping().apply(self.basic_data).serialize()
+        strict = SomeStrictMapping().apply(self.basic_data).serialize()
+
+        self.assertDictEqual(self.non_strict_output, non_strict)
+        self.assertDictEqual(self.strict_output, strict)
+
+    def test_all_transformation_generic_schema(self):
+
+        class SomeNonStrictMapping(Mapping):
+            user_name = Get('user', 'name')
+            object = All(strict=False)
+
+        class SomeStrictMapping(Mapping):
+            user_name = Get('user', 'name')
+            object = All(strict=True)
+
+        non_strict = SomeNonStrictMapping().apply(self.basic_data).serialize()
```

Comment:
I understand you're trying to be DRY but putting the fixtures elsewhere like this feels like a loss for clarity. I can't understand this test at a glance.